source/blog/2018-03-26-ember-3-1-released.md

```diff
@@ -0,0 +1,149 @@
+---
+title: Ember 3.1 and 3.2 Beta Released
+author: Ricardo Mendes
+tags: Releases
+---
+
+Today the Ember project is releasing version 3.1.0 of Ember.js, Ember Data, and Ember CLI.
+
+This release kicks off the 3.2 beta cycle for all sub-projects. We encourage our
+community (especially addon authors) to help test these beta builds and report
+any bugs before they are published as a final release in six weeks' time. The
+[ember-try](https://github.com/ember-cli/ember-try) addon is a great way to
+continuously test your projects against the latest Ember releases.
+
+You can read more about our general release process here:
+
+- [Release Dashboard](http://emberjs.com/builds/)
+- [The Ember Release Cycle](http://emberjs.com/blog/2013/09/06/new-ember-release-process.html)
+- [The Ember Project](http://emberjs.com/blog/2015/06/16/ember-project-at-2-0.html)
+- [Ember LTS Releases](http://emberjs.com/blog/2016/02/25/announcing-embers-first-lts.html)
+
+---
+
+## Ember.js
+
+Ember.js is the core framework for building ambitious web applications.
+
+### Changes in Ember.js 3.1
+
+Ember.js 3.1 is an incremental, backwards compatible release of Ember with
+bugfixes, performance improvements, and minor deprecations.
+
+#### Deprecations in Ember 3.1
+
+Deprecations are added to Ember.js when an API will be removed at a later date.
+
+Each deprecation has an entry in the deprecation guide describing the migration
+path to more stable API. Deprecated public APIs are not removed until a major
+release of the framework.
+
+Consider using the
+[ember-cli-deprecation-workflow](https://github.com/mixonic/ember-cli-deprecation-workflow)
+addon if you would like to upgrade your application without immediately addressing
+deprecations.
+
+Two new deprecations are introduces in Ember.js 3.1:
+
+* TODO
+* TODO
+
+For more details on changes in Ember.js 3.1, please review the
+[Ember.js 3.1.0 release page](https://github.com/emberjs/ember.js/releases/tag/v3.1.0).
+
+### Upcoming Changes in Ember.js 3.2
+
+Ember.js 3.2 will introduce two new features:
+
+* TODO
+* TODO
+
+#### Deprecations in Ember.js 3.2
+
+Two new deprecations are introduces in Ember.js 3.2:
+
+* TODO
+* TODO
+
+For more details on the upcoming changes in Ember.js 3.2, please review the
+[Ember.js 3.2.0-beta.1 release page](https://github.com/emberjs/ember.js/releases/tag/v3.2.0-beta.1).
+
+---
+
+## Ember Data
+
+Ember Data is the official data persistence library for Ember.js applications.
+
+### Changes in Ember Data 3.1
+
+#### Deprecations in Ember Data 3.1
+
+Two new deprecations are introduces in Ember Data 3.1:
+
+* TODO
+* TODO
+
+For more details on changes in Ember Data 3.1, please review the
+[Ember Data 3.1.0 release page](https://github.com/emberjs/data/releases/tag/v3.1.0).
+
+
+### Upcoming changes in Ember Data 3.2
+
```

Comment:
[#5230](https://github.com/emberjs/data/pull/5230) [BUGFIX] enable lazy-relationship payloads to work with polymorphic relationships